RxSwiftWorkshopTests/Utilities/Equatability.swift

```diff
@@ -0,0 +1,13 @@
+//
+// Created by Krzysztof Siejkowski on 03/05/2017.
+// Copyright (c) 2017 Mobile Academy. All rights reserved.
+//
+
+import Foundation
+import CoreLocation
+
+extension CLLocationCoordinate2D: Equatable {
```

Comment:
That's interesting.
I'd growth creating file per each `Equatable` protocol adoption, e.g. `CLLocationCoordinate2D+Equatable` in this case, or `CLLocationCoordinate2D+Extensions` at least.

But I'm not super tight to this, maybe it is not very good idea.